app/views/questions/show.html.erb

```diff
@@ -85,7 +82,7 @@
                 <%= f.hidden_field :question, value: params[:id] %>
```

Comment:
This breaks if the user had just submitted an empty comment and got an error, since params will no longer hold the question id.